### ISSUE TITLE
Minor: CI build badge link not working

### ISSUE THREAD

**Issue by bhamail** [sonatype-nexus-community/bach#41]
The current CI badge, when clicked, only displays an image of the build badge itself, rather than the actual CI build pages.

Here's one alternative to consider if the shields.io one proves troublesome:

```
[![sonatype-nexus-community](https://circleci.com/gh/sonatype-nexus-community/bach.svg?style=shield)](https://circleci.com/gh/sonatype-nexus-community/bach)
```

**Comment by madpah** [comment]
Thanks @bhamail - should be fixed on `main` now if you're in agreement?

**Comment by bhamail** [comment]
Fixed by 2a9a265475ad6dd5cb673ca50cf1c3106923f851
